src/AppInstallerCLITests/Correlation.cpp

```diff
@@ -242,10 +242,10 @@ DataSet GetDataSet_WithNoise()
     dataSet.TestCases = std::move(baseTestCases);
 
     // Arbitrary values. We should refine them as the algorithm gets better.
-    dataSet.RequiredTrueMatchRatio = 0.5;
-    dataSet.RequiredFalseMatchRatio = 0.1;
+    dataSet.RequiredTrueMatchRatio = 0.7;
+    dataSet.RequiredFalseMatchRatio = 0.05;

```

Comment:
I would really like false positives to be at 0.  If the current algorithm is not at 0 in these unit tests, we should reconsider.

Reply:
I'll send another PR once I figure out how to reduce the false positives.